main/views.py

```diff
@@ -40,16 +40,14 @@ class ProjectsListView(LoginRequiredMixin, SingleTableMixin, FilterView):
     filterset_fields = ("nature", "department", "status", "charging")
 
 
-class ProjectDetailView(LoginRequiredMixin, UpdateView):  # type: ignore [type-arg]
+class CustomBaseDetailView(LoginRequiredMixin, UpdateView):  # type: ignore [type-arg]
```

Comment:
I made the base class as the views seem to be basically the same except for the heading at the top. Should the base class be in a separate file?